paymentsheet/src/main/java/com/stripe/android/paymentsheet/addresselement/AddressElementActivity.kt

```diff
@@ -127,11 +121,3 @@ internal class AddressElementActivity : ComponentActivity() {
         overridePendingTransition(AnimationConstants.FADE_IN, AnimationConstants.FADE_OUT)
     }
 }
-
-private fun NavHostController.navigateToContent() {
-    return navigate(AddressElementScreen.InputAddress.route) {
```

Comment:
Removed this as changing navigation this early was causing issues with the tests.